opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -1237,6 +1289,17 @@ namespace Opm {
             well->assembleWellEq(ebosSimulator_, dt, this->wellState(), this->groupState(), deferred_logger);
         }
     }
+    template<typename TypeTag>
```

Comment:
Any reason to not separate this function from its predecessor by at least one blank line?